pyproject.toml

```diff
@@ -22,7 +22,7 @@ classifiers = [
     "Programming Language :: Python :: 3.10",
 ]
 requires-python = ">=3.7.0"
-dependencies = ["lark>=0.10.0,<0.11.0"]
+dependencies = ["lark>=0.10.0"]
```

Comment:
```suggestion
dependencies = ["lark>=0.10.0,<2"]
```

Reply:
I believe even better to set it to `dependencies = ["lark>=1,<2"]` 
We will release a new version with this new requirement